code/game/machinery/computer/communications.dm

```diff
@@ -390,7 +390,7 @@
 
 					data["alertLevelTick"] = alert_level_tick
 					data["canMakeAnnouncement"] = TRUE
-					data["canMakeVoiceAnnouncement"] = ishuman(user)
+					data["canMakeVoiceAnnouncement"] = ishuman(user) || IsAdminGhost(user)
```

Comment:
Why not move this up to the authenticated_as_non_silicon_captain block, and set it to true?